tests/wpt/tests/css/css-display/display-flow-root-text-control.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Since we are referring to the HTML spec, this test should go to `html/rendering/widgets/`, which is the same as the chapter titles of the spec.